<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnivariateAnalysisDialog</class>
 <widget class="QDialog" name="UnivariateAnalysisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Wavelength Range</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="peakComboBox">
     <item>
      <property name="text">
       <string>Intensity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Area</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bandwidth</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Area Ratio</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Intensity Ratio</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Region 1</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="firstMinLineEdit_2">
       <property name="placeholderText">
        <string>Region 1 Left Bound</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="firstMaxLineEdit_2">
       <property name="placeholderText">
        <string>Region 1 Right Bound</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="secondMinLineEdit_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Region 2 Left Bound</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="secondMaxLineEdit_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Region 2 Right Bound</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Value Determination</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="rangeLabel_2">
     <property name="text">
      <string>200-2000 cm-1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="integrationLabel">
     <property name="text">
      <string>Integration Method</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Region 2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="integrationComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Riemann Sum</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UnivariateAnalysisDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UnivariateAnalysisDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
